components/webui/terraform/build.tf

```diff
@@ -14,23 +14,29 @@
 
 locals {
   ui_service_name     = "eks-ui"
-  cloud_build_fileset = setunion(fileset(path.module, "../src/**"), fileset(path.module, "../Dockerfile"), fileset(path.module, "../requirements.txt"))
+  cloud_build_fileset = setunion(fileset(path.module, "../src/**"), fileset(path.module, "build/Dockerfile"), fileset(path.module, "build/requirements.txt"), fileset(path.module, "build/cloudbuild.yaml"))
   cloud_build_content_hash = sha512(join(",", [
   for f in local.cloud_build_fileset : fileexists("${path.module}/${f}") ? filesha512("${path.module}/${f}") : sha512("file-not-found")]))
 }
 
+resource "local_file" "cloudbuild_config" {
+  filename = "${path.module}/build/cloudbuild.yaml"
+  content = templatefile("${path.module}/build/cloudbuild.yaml.template", {
+    project_id            = var.project_id,
+    build_service_account = var.cloud_build_service_account_email,
+    image_tag             = "${var.region}-docker.pkg.dev/${module.project_services.project_id}/${var.artifact_repo}/${local.ui_service_name}"
+  })
+}
+
 # Build and upload the app container
 module "app_build" {
-  source          = "github.com/terraform-google-modules/terraform-google-gcloud?ref=db25ab9c0e9f2034e45b0034f8edb473dde3e4ff" # commit hash of version 3.5.0
-  platform        = "linux"
-  create_cmd_body = "builds submit --region ${var.region} --project ${var.project_id} --tag \"${var.region}-docker.pkg.dev/${module.project_services.project_id}/${var.artifact_repo}/${local.ui_service_name}\" \"${path.module}/../\""
-  enabled         = true
+  source = "github.com/terraform-google-modules/terraform-google-gcloud?ref=db25ab9c0e9f2034e45b0034f8edb473dde3e4ff" # commit hash of version 3.5.0
+
+  create_cmd_entrypoint = "gcloud"
+  create_cmd_body       = "builds submit --region ${var.region} --project ${var.project_id} --config \"${local_file.cloudbuild_config.filename}\" \"${path.module}/../../..\""
```

Comment:
nit: can you break this up into multiple lines using EOT, like how it's done in this file?
https://github.com/GoogleCloudPlatform/enterprise-knowledge-solution/blob/1ce4284d3e5025829c2ab37da83b2b6794abfb76/components/doc-classifier/terraform/build.tf#L27

Just easier to read that way

Reply:
And, there's one other instance of `create_cmd_body` in enterprise-knowledge-solution/components/processing/terraform/build.tf, can you tweak to the same EOT patterns for consistency?

Reply:
done in both places